### PR TITLE
Sharing layout fixes

### PR DESCRIPTION
**Pull request by jancborchardt**
Before:
capture du 2016-11-15 12-53-47
After:
capture du 2016-11-15 12-53-19

Please review @nextcloud/designers @nextcloud/sharing

### CODE REVIEW

**Comment by jancborchardt** [comment]
Also cc for review @juliabode @hechi because you worked on the sharing sidebar. :)

**Comment by rullzer** [comment]
Looks good :+1:

File: apps/files_sharing/css/sharetabview.css [hunk 1]
```diff
@@ -54,38 +54,45 @@
...
 	margin-left: 0;
 	position: relative;
 }
-#shareWithList .username{
+#shareWithList .username {
 	padding-right: 8px;
 	white-space: nowrap;
 	text-overflow: ellipsis;
-	max-width: 254px;
+	max-width: 110px !important;
```
Comment (MorrisJobke) [inline apps/files_sharing/css/sharetabview.css]: Setting this to `80px` also fixes it for german:

bildschirmfoto 2016-11-17 um 00 11 29
bildschirmfoto 2016-11-17 um 00 11 37

Comment (MorrisJobke) [inline apps/files_sharing/css/sharetabview.css]: But this only applies for screens smaller than ~1100px

**Comment by MorrisJobke** [comment]
Beside that it looks good - @jancborchardt could you add this for to the mobile.css?

**Comment by jancborchardt** [comment]
Fixed, please review again @MorrisJobke @rullzer @schiessle  @nextcloud/designers  :)

File: apps/files/css/mobile.css [hunk 2]
```diff
@@ -61,9 +61,9 @@ table td.filename .nametext .innernametext {
 	max-width: 50%;
 }
 
-/* ellipsis on user names in share sidebar */
+/* ellipsis on user names in share sidebar, less on mobile */
 #shareWithList .username {
-    max-width: 80px !important;
+	max-width: 80px !important;
```
Comment (MorrisJobke) [inline apps/files/css/mobile.css]: This is still not used, because the one in `sharetabview.css` also has the `!important`

**Review by MorrisJobke — DISMISSED** [review]

**Comment by MorrisJobke** [comment]
cc @icewind1991

**Comment by icewind1991** [comment]
:+1: tested, works

I would still like (for a separate PR) to have the `...` aligned with the input field+help icon above it

**Comment by jancborchardt** [comment]
@icewind1991 fixed it, and more goodness. Please review @nextcloud/designers  :)

**Comment by icewind1991** [comment]
:+1:

**Pull request merged**